extensions/sql-migration/src/service/contracts.ts

```diff
@@ -136,6 +137,7 @@ export interface AzureManagedDiskSku {
 	tier: AzureManagedDiskTier;
 	size: string;
 	caching: AzureManagedDiskCaching;
+	type: AzureManagedDiskType;
```

Comment:
let's the 3 new size related properties as well :)

Reply:
Done. Thanks for adding context from NuGet implementation.